tests/conftest.py

```diff
@@ -216,3 +217,21 @@ def pytest_configure(config):
     config.dvc_config.dvc_git_repo = config.getoption("--dvc-git-repo")
     config.dvc_config.project_rev = config.getoption("--project-rev")
     config.dvc_config.project_git_repo = config.getoption("--project-git-repo")
+
+    remotes = set(enabled_remotes)
+    if config.dvc_config.remote != "local":
+        remotes.add(config.dvc_config.remote)
+
+    for remote in remotes:
+        try:
+            module = importlib.import_module(f"dvc_{remote}.tests.fixtures")
+            globals().update(
+                (k, v)
+                for k, v in module.__dict__.items()
+                if not k.startswith("_")
+            )
```

Comment:
feels like this should really be handled by registering an appropriate entrypoint in each plugin (whether we make it some `dvc.testing` entrypoint or just register the fixtures via `pytest` entrypoints)